sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -256,8 +256,19 @@ public abstract partial class Response : System.IDisposable
     public sealed partial class ResponseError
     {
         public ResponseError(string? code, string? message) { }
+        public ResponseError(string? code, string? message, string? target, Azure.ResponseInnerError? innerError = null, System.Collections.Generic.IReadOnlyList<Azure.ResponseError>? details = null) { }
         public string? Code { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResponseError> Details { get { throw null; } }
```

Comment:
Is this ever used?  In the issue, it sounds like KV only needs the inner error?

Reply:
Lots of services return details e.g., some Cognitive. See my comment above about the other error type for a pointer.